### PR TITLE
Changed method of getting local BOOTSTRAP_SERVERS

### PR DESCRIPTION
**Pull request by sjwoodman**
Constructs the value from the name of a Strimzi CR

### CODE REVIEW

File: operator/src/main/java/io/streamzi/eventflow/ClientCache.java [hunk 1]
```diff
@@ -34,9 +37,27 @@
...
+                    kafkaCRD,
+                    Kafka.class,
+                    KafkaAssemblyList.class,
+                    DoneableKafka.class)
+                    .inNamespace(osClient.getNamespace())
+                    .list()
+                    .getItems();
+
+            if (kafkas.size() > 0) {
+                String localBs = kafkas.get(0).getMetadata().getName() + "-kafka-bootstrap." + kafkas.get(0).getMetadata().getNamespace() + ".svc.cluster.local:9092";
```
Comment (matzew) [inline operator/src/main/java/io/streamzi/eventflow/ClientCache.java]: `localBs` ? :-)

File: operator/src/main/java/io/streamzi/eventflow/ClientCache.java [hunk 2]
```diff
@@ -34,9 +37,27 @@
...
+                logger.info("Setting BOOTSTRAP_SERVERS for local to " + localBs);
+                bootstrapServerCache.put("local", localBs);
```
Comment (matzew) [inline operator/src/main/java/io/streamzi/eventflow/ClientCache.java]: cache is currently only a simple map, right ?

File: operator/src/main/java/io/streamzi/eventflow/ClientCache.java (hunk 1 above)
Comment (sjwoodman) [reply operator/src/main/java/io/streamzi/eventflow/ClientCache.java]: :)

File: operator/src/main/java/io/streamzi/eventflow/ClientCache.java (hunk 2 above)
Comment (sjwoodman) [reply operator/src/main/java/io/streamzi/eventflow/ClientCache.java]: Yes, is that ok?

Comment (matzew) [reply operator/src/main/java/io/streamzi/eventflow/ClientCache.java]: > Yes, is that ok?

yes, we just need to have that in mind - I have a history w/ _improving_ caches :)

**Pull request merged**